parm/config/gefs/config.wave

```diff
@@ -23,6 +23,16 @@ export waveMULTIGRID='.false.'        # .true./.false. for multi or shel
 export MESH_WAV="mesh.${waveGRD}.nc"  # Mesh grid for wave model for CMEPS
 export waveesmfGRD=' '                # input grid for multigrid
 
+export WW3_user_histname="false"
+export WW3_historync="false"
+export WW3_restartnc="true"
+export WW3_PIO_FORMAT="pnetcdf"
+export WW3_PIO_IOTASKS=-99
+export WW3_PIO_STRIDE=4
+export WW3_PIO_REARR="box"
+export WW3_PIO_ROOT=-99
+
+
```

Comment:
Can I ask the motivation to duplicate this information in `config.wave` for `gfs` and `gefs` rather than keeping it in `parsing_namelists_WW3.sh`.  Is this expected to be configurable by a user?

Reply:
Just for these variables or in general? 

For these variables: 
These PIO variables could likely just be put in the parsing_ufs.configure file to reduce duplication. I think we originally thought they'd be different for gefs/gfs but it is not.

Reply:
These ones came to the front in this PR, but in general it is up to you and the team.  If things need to be configurable by a user or common between tasks, then we should make room for it in `config.wave`, otherwise we could keep them local to the function where necessary.  
Nothing hard or fast here, and just trying to keep things simpler/cleaner

Reply:
I haven't thought about this in a while, but it's a good thought.  I'll probably keep things as is for right now if that's okay but discuss with @sbanihash to see if there's way to simplify/reduce duplication moving forward.

Reply:
works for me.